src/material/button/_fab-theme.scss

```diff
@@ -123,15 +123,15 @@
     @include _theme-from-tokens(inspection.get-theme-tokens($theme, typography));
   } @else {
     @include sass-utils.current-selector-or-root() {
-      @include token-utils.create-token-values(
+      @include token-utils.create-token-values-mixed(
```

Comment:
Should we come up some more explicit name for this function? We can do it in a separate PR.

Reply:
Yeah, we can bikeshed another name. The opposite case, `-inline`, would be an easy name but I dont know what `not-inline` would be